packages/cli/src/cmds/validator/signers/index.ts

```diff
@@ -91,7 +91,11 @@ export async function getSignersFromArgs(
         );
       },
     });
-    return await decryptKeystoreDefinitions(keystoreDefinitions, {...args, onDecrypt: needle});
+    return decryptKeystoreDefinitions(keystoreDefinitions, {
+      ...args,
+      onDecrypt: needle,
+      cacheFilePath: `${args.importKeystores[0]}.cache`,
```

Comment:
@nazarhussain I noticed that the `validator_keys.cache` is stored in the same directory as my `validator_keys` folder which I set via `--importKeystores`. This is a not ideal in a containerized environment as the cache file is likely not mounted to the container or persistent in a volume. Maybe it would be better to store the cache file in the data directory (`dataDir`).

In our `docker-compose.validator.yml` file we also only mount the `/keystores` which would not include the cache file.

https://github.com/ChainSafe/lodestar/blob/2042c3b1c1916ceed6bc825057515a4456c33a08/docker-compose.validator.yml#L4-L11